docs/how-to-use/search/index.md

```diff
@@ -14,7 +14,7 @@ DspSearchModule is part of @dasch-swiss/dsp-ui, follow [the installation guide](
 
 ## Components
 
-This module contains various components to search. The main component is the dsp-search-panel, which contains the dsp-fulltext-search, dsp-extended-search and dsp-expert-search. All of them can be used standalone or in combination in dsp-search-panel.
+This module contains various components to search. The main component is the dsp-search-panel, which contains the dsp-fulltext-search, dsp-advanced-search and dsp-expert-search. All of them can be used standalone or in combination in dsp-search-panel.
```

Comment:
standalone -> individually (standalone is correct and understandable, I would just personally use individually)